docs/fundamentals/code-analysis/quality-rules/singlefile-warnings.md

```diff
@@ -1,20 +1,20 @@
 ---
-title: Publish rules (code analysis)
-description: "Learn about code analysis publish rules."
+title: SingleFile rules (code analysis)
+description: "Learn about code analysis single file rules."
 ms.date: 09/04/2020
 ms.topic: reference
 f1_keywords:
-- vs.codeanalysis.publishrules
+- vs.codeanalysis.singlefilerules
 helpviewer_keywords:
-- rules, publish
-- managed code analysis rules, publish rules
-- publish rules
+- rules, singlefile
+- managed code analysis rules, singlefile rules
+- singlefile rules
 author: gewarren
 ms.author: gewarren
 ---
-# Publish rules
+# SingleFile rules
 
-Publish rules for .NET.
+SingleFile rules for .NET.
```

Comment:
```suggestion
SingleFile rules for .NET.

> [!NOTE]
> Prior to .NET 6, this category was named `Publish`.
```

Reply:
@gewarren Individual rule pages doesn't have a link to *singlefile-warnings.md*. This makes it more likely that page visitors won't see the note. I suggest either adding a link to singlefile-warnings.md under a "See also" section, or have the note in the individual pages too.

Can be done as a follow-up if you prefer.

Reply:
As a follow up, we should add a similar table to all 3 singlefile rules:

![image](https://user-images.githubusercontent.com/24882762/115916330-87916800-a429-11eb-8aff-316c526049c6.png)

Reply:
Opened https://github.com/dotnet/docs/issues/23910 so this is not forgotten.